test/JDBC/expected/BABEL-2403.out

```diff
@@ -81,6 +81,8 @@ name#!#pg_catalog#!#rolname#!#{"Rule": "<rolname> in babelfish_authid_login_ext
 text#!#sys#!#name#!#{"Rule": "<default_database_name> in babelfish_authid_login_ext must also exist in babelfish_sysdatabases"}
 name#!#pg_catalog#!#rolname#!#{"Rule": "<rolname> in babelfish_authid_login_ext must also exist in pg_authid"}
 text#!#sys#!#name#!#{"Rule": "<default_database_name> in babelfish_authid_login_ext must also exist in babelfish_sysdatabases"}
+name#!#pg_catalog#!#rolname#!#{"Rule": "<rolname> in babelfish_authid_login_ext must also exist in pg_authid"}
+text#!#sys#!#name#!#{"Rule": "<default_database_name> in babelfish_authid_login_ext must also exist in babelfish_sysdatabases"}
```

Comment:
Why are these 2 entries appearing?

Reply:
Because we have added one additonal login for securityadmin in the catalog.